contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/checker.py

```diff
@@ -14,6 +14,7 @@
 from pants.option.custom_types import file_option
 from pants.task.lint_task_mixin import LintTaskMixin
 from pants.task.task import Task
+from pex.interpreter import PythonInterpreter
```

Comment:
This should be paired with a `'3rdparty/python:pex'` dependency in `contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/BUILD`.